.github/workflows/publish-chocolatey-package.yml

```diff
@@ -0,0 +1,82 @@
+name: Update Chocolatey Package
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to use (optional, defaults to latest release)'
+        required: false
+  release:
+    types: [published]
+
+jobs:
+  update-choco:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Determine tag to use
+        id: determine-tag
+        run: |
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" && "${{ github.event.inputs.tag }}" != "" ]]; then
+            echo "TAG=${{ github.event.inputs.tag }}" >> "$GITHUB_OUTPUT"
+          elif [[ "${{ github.event_name }}" == "release" ]]; then
+            echo "TAG=${{ github.event.release.tag_name }}" >> "$GITHUB_OUTPUT"
+          else
+            latest_tag=$$(git describe --tags --abbrev=0 --match "v*")
+            echo "TAG=$latest_tag" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Checkout code at specified tag
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ steps.determine-tag.outputs.TAG }}
+          fetch-depth: 0
+
+      - name: Install Mono & dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y mono-complete wget tar jq
```

Comment:
the following packages are part of the Ubuntu image (https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md):

- jq
- wget
- tar

We only need mono-complete

Reply:
Good point